### PR TITLE
Fix line_separation working incorrectly in find_click of RichTextLabel

### PR DESCRIPTION
**Pull request by requizm**
Related #49455 #51866

https://github.com/godotengine/godot/pull/51866#discussion_r693229211

### CODE REVIEW

**Comment by Calinou** [comment]
cc @bruvzg

File: scene/gui/rich_text_label.cpp [hunk 1]
```diff
@@ -1127,7 +1127,7 @@ void RichTextLabel::_find_click(ItemFrame *p_frame, const Point2i &p_click, Item
 	Point2 ofs = text_rect.get_position() + Vector2(0, main->lines[from_line].offset.y - vofs);
 	while (ofs.y < size.height && from_line < main->lines.size()) {
 		_find_click_in_line(p_frame, from_line, ofs, text_rect.size.x, p_click, r_click_frame, r_click_line, r_click_item, r_click_char);
-		ofs.y += main->lines[from_line].text_buf->get_size().y;
+		ofs.y += main->lines[from_line].text_buf->get_size().y + +get_theme_constant(SNAME("line_separation"));
```
Comment (bruvzg) [inline scene/gui/rich_text_label.cpp:1130]:
```suggestion
		ofs.y += main->lines[from_line].text_buf->get_size().y + get_theme_constant(SNAME("line_separation"));
```
Extra "+"

Comment (requizm) [reply scene/gui/rich_text_label.cpp:1130]: Oh sorry, i rushed  for a reason.

**Review by bruvzg — APPROVED** [review]

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**